internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+AWSTemplateFormatVersion: 2010-09-09
+Description: CloudFormation template that represents a static site stored in S3.
```

Comment:
An alternative is `Description: CloudFormation template that represents a static site backed by S3.`. I don't have preference though!